server/server.go

```diff
@@ -1175,23 +1203,25 @@ func createRouter(eng *engine.Engine, logging, enableCors bool, corsHeaders stri
 			"/version":        getVersion,
 		},
 		"POST": {
-			"/auth":             postAuth,
-			"/attach":           postAttach,
-			"/container/create": postContainerCreate,
-			"/container/commit": postContainerCommit,
-			"/container/rename": postContainerRename,
-			"/exec":             postExec,
-			"/image/create":     postImageCreate,
-			"/image/build":      postImageBuild,
-			"/image/push":       postImagePush,
-			"/pod/create":       postPodCreate,
-			"/pod/labels":       postPodLabels,
-			"/pod/start":        postPodStart,
-			"/pod/stop":         postStop,
-			"/service/add":      postServiceAdd,
-			"/service/update":   postServiceUpdate,
-			"/tty/resize":       postTtyResize,
-			"/vm/create":        postVmCreate,
+			"/auth":              postAuth,
+			"/attach":            postAttach,
+			"/container/create":  postContainerCreate,
+			"/container/commit":  postContainerCommit,
+			"/container/rename":  postContainerRename,
+			"/container/pause":   postContainerPause,
+			"/container/unpause": postContainerUnpause,
```

Comment:
@laijs pause api introduced here is for docker commit container, commit need to pause container. we provide this api to consist with docker api. the internal implements can be change when we can really implement pause container.